inst/rmarkdown/templates/gigascience_article/skeleton/paper-refs.bib

```diff
@@ -0,0 +1,130 @@
+@Book{Anderberg83,
+   author =   {Anderberg, M. R.},
+   title =    {Cluster Analysis for Applications},
+   publisher =    {New York: Academic Press},
+   year =     {1983},
+ }
+
+  @Manual{R:2010,
+    title = {R: A Language and Environment for Statistical Computing},
+    author = {{R Development Core Team}},
+    organization = {Vienna, Austria: R Foundation for Statistical Computing},
+    address = {},
+    year = {2012},
+    note = {{ISBN} 3-900051-07-0, http://www.R-project.org},
+    url = {http://www.R-project.org},
+  }
+
+  @Manual{R:2008,
+    title = {R: A Language and Environment for Statistical Computing},
+    author = {{R Development Core Team}},
+    organization = {R Foundation for Statistical Computing},
+    address = {Vienna, Austria},
+    year = {2008},
+    note = {{ISBN} 3-900051-07-0},
+    url = {http://www.R-project.org},
+  }
```

Comment:
Do we need two entries for R?

Could you use the bib entry for the latest version of R?

And it will be great if only those bib entries that are actually cited in the skeleton.Rmd are kept in this bib file.

Reply:
I absolutely *don't** need two bib entries. :) I will fix this.